### PR TITLE
[ARM] az deployment-scripts: Removed preview flag

### PR DESCRIPTION
**Pull request by jorgecotillo**
**Description**
This PR removes the preview flag from az deployment-scripts commands, and uses the latest api-version which is 2020-10-01.

**Testing Guide**
Not much changes from before, just added a create and read corresponding to 2020-10-01 api-version.

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ARM

**Comment by jorgecotillo** [comment]
@zhoxing-ms can you please help us review this PR? Is needed for our service to go GA. Thanks!

File: src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py [hunk 1]
```diff
@@ -1653,7 +1653,7 @@ def test_subscription_level_what_if_ts(self, resource_group, resource_group_loca
 
 
 class DeploymentScriptsTest(ScenarioTest):
-    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts')
+    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts', location='brazilsouth')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py:1656]: > location='brazilsouth'

May I ask why you added location in those test?

Comment (jorgecotillo) [reply src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py:1656]: Sure, there's a known issue in ARM that when a service endpoint has multiple locations (e.g. one endpoint that serves east us and west us), ARM returns duplicated resources during a List operation, in our case, brazil south has one endpoint and one location.

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py:1656]: Got it~ Thanks

**Review by zhoxing-ms — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**